src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+enum Color {
+    BLUE,
+    RED,
+    GREEN
+}
+
+public class Ball {
+    private String color;
```

Comment:
The `color` field should be of type `Color`, not `String`, as it represents an enum of colors.